core/src/main/java/io/opencensus/tags/TagContext.java

```diff
@@ -31,17 +32,17 @@
  */
 @Immutable
 public final class TagContext {
-  /** The maximum length for a string tag value. */
-  public static final int MAX_STRING_LENGTH = StringUtil.MAX_LENGTH;
+
+  private static final TagContext EMPTY = newBuilder().build();
```

Comment:
Correct. Collections.emptyList() has to do a cast so that is one of the reason to not have it directly as value.